lib/assigner/CMakeLists.txt

```diff
@@ -5,19 +5,34 @@ cmake_policy(SET CMP0063 NEW)
 
 option(BUILD_ASSIGNER_TESTS "Build unit tests" FALSE)
 
-add_library(${PROJECT_NAME} STATIC src/vm_host.cpp src/handler_base.cpp)
+set(evmone_sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/../evmone/baseline.cpp
```

Comment:
Not sure. Actually i prefer keep in the separate folders  logic responsible for filling assignment  tables and op_code processing.
May be make sense move vm_host.h to evmone and rename to vm_host.hpp

Reply:
Maybe at least avoid accessing parent directory here? It is uncommon in cmake and makes it difficult to understand project structure. We could either move evmone folder inside asiigner folder or move this cmake code into upper CMakeLists.txt. 
I think the first approach is better, because now we have two directories inside `lib/`, but actually only one library.